docs.bruno/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "API_Testing",
+  "type": "collection"
+}
\ No newline at end of file
```

Comment:
It's a good practice to add a newline at the end of the file to avoid any potential issues with certain tools that might require it. [medium]